utils/oscap.8

```diff
@@ -100,6 +100,11 @@ Write XCCDF results into FILE.
 Writes results to a given FILE in Asset Reporting Format. It is recommended to use this option instead of --results when dealing with datastreams.
 .RE
 .TP
+\fB\-\-without-syschar\fR
+.RS
+Don't provide system characteristics in result file.
```

Comment:
Oops, indeed it was kinda misleading, added a more descriptive text for this option.